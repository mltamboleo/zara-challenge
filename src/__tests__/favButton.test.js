import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import FavButton from '../app/components/favButton/index.tsx'

jest.mock('next/router', () => ({
  useRouter() {},
}));

describe('Fav Button Component', () => {
  it('Renders Fav Button', () => {
    const { container } = render(<FavButton />)
    expect(container).toMatchSnapshot()
  })
})
