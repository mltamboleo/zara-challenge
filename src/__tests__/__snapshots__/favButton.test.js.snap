// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fav Button Component Renders Fav Button 1`] = `
<div>
  <button
    class="fav-button undefined"
  >
    <img
      alt="Add to favorites"
      data-nimg="1"
      decoding="async"
      height="12"
      loading="lazy"
      src="/fav-unselected.svg"
      style="color: transparent;"
      width="12"
    />
  </button>
</div>
`;
